src/test/java/com/whiletrue/tododemo/controller/TaskControllerTest.java

```diff
@@ -0,0 +1,106 @@
+package com.whiletrue.tododemo.controller;
+
+import com.whiletrue.tododemo.dto.TaskRequest;
+import com.whiletrue.tododemo.dto.UserRequest;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.specification.RequestSpecification;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.server.LocalServerPort;
+
+import javax.annotation.PostConstruct;
+import java.time.Instant;
+import java.util.List;
+
+import static io.restassured.RestAssured.with;
+import static org.hamcrest.Matchers.equalTo;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
```

Comment:
You don't need that. Actually, just remove it.
Tests should not depend on each other. Each test should test one thing.
For example: an update task test should first prepare a mocked task and then update it.